src/NetscriptFunctions/Bladeburner.ts

```diff
@@ -117,10 +132,20 @@ export function NetscriptBladeburner(): InternalAPI<INetscriptBladeburner> {
         1000
       );
     },
-    getActionEstimatedSuccessChance: (ctx) => (type, name) => {
+    getActionEstimatedSuccessChance: (ctx) => (type, name, _sleeve) => {
       const bladeburner = getBladeburner(ctx);
       const action = getAction(ctx, type, name);
-      return action.getSuccessRange(bladeburner, Player);
+      if (_sleeve !== undefined) {
```

Comment:
I think I'd check against `!= null` here to work with both `undefined` and `null`, in case someone passes `null` directly.